src/Sulu/Bundle/MediaBundle/Resources/config/doctrine/CollectionMeta.orm.xml

```diff
@@ -4,7 +4,6 @@
                   xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
     <entity name="Sulu\Bundle\MediaBundle\Entity\CollectionMeta" table="me_collection_meta">
         <indexes>
-            <index columns="title"/>
```

Comment:
not sure which fields we should set to 191 or remove from being indexed. /cc @chirimoya @danrot

Reply:
Are you completely sure that the title is never queried with `value%`? Because that would work with an index. I think I would leave it there just to be save.

Reply:
Currently in the repository always with `%` prefix. And I think the field descriptors search do the same?
But we can let the index and set the field length 191 characters.